src/main/java/pl/sda/partyka/DataInitializer.java

```diff
@@ -0,0 +1,21 @@
+package pl.sda.partyka;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.stereotype.Component;
+import pl.sda.partyka.domain.Role;
+import pl.sda.partyka.service.RoleService;
+
+@Component
+@RequiredArgsConstructor
+public class DataInitializer implements ApplicationRunner {
```

Comment:
Are you sure if this is a part of the production code? Maybe you should use database initializer https://stackoverflow.com/a/38047021/4414894